sled-agent/src/rack_setup/service.rs

```diff
@@ -123,14 +191,9 @@ impl RackSetupService {
     }
 }
 
-fn rss_plan_path() -> std::path::PathBuf {
+fn rss_completed_plan_path() -> PathBuf {
```

Comment:
Tiny nit - based on the name I expected this to return `rss-plan-complete.toml`, not `.marker`. Maybe `rss_plan_executed_path()` or `rss_completed_marker_path()` would be clearer?

Reply:
Done in d223fb9b1798e03ae7debdebacc9cdfaa8344666